### ISSUE TITLE
ImportModel prompts a SystemCheckError

### ISSUE THREAD

**Issue by Ezra** [edcrewe/django-csvimport#100]
Pursuant to #85, the current fix for `on_delete` prompts `manage.py check` to give an error.

To reproduce:
1.  follow the "demo" steps up through creating 'django-admin.py'
2. `python django-admin.py check`

Expected:
`System check identified no issues (0 silenced).`
Actual:
```
SystemCheckError: System check identified some issues:

ERRORS:
csvimport.ImportModel.csvimport: (fields.E312) The to_field 'True' doesn't exist on the related model 'csvimport.CSVImport'.

System check identified 1 issue (0 silenced).
```

This error can be traced to the line
`csvimport = models.ForeignKey(CSVImport, on_delete=True)`
which was introduced in 05c3a85. `True` is not a valid `on_delete` handler.

The correct code exists identically in both the pull requests #91 and #92

**Comment by edcrewe** [comment]
Fixed in 2.15

**Comment by Ezra** [comment]
Looks great, thank you!
